packaging/server_config.json.template

```diff
@@ -28,6 +28,12 @@
     "default_hash_scheme": "SHA256",
     "environment_variables": {},
     "federation": [],
+    "host_access_control": {
+        "access_entries": []
+    },
+    "host_resolution": {
+        "host_entries": []
+    },
```

Comment:
@trel @alanking @SwooshyCueb How do we feel about **host_resolution** representing the **hosts_config.json** file?

Reply:
i think it works... any other top candidates?      this is the iRODS-level equivalent of `/etc/hosts`.

we can also change the format of the entries itself during this migration / redefinition of the schemas.   i remember we were not too happy with the inherited format.   could be much easier to read/process...

Reply:
we didn't make one - just the existing one was a bit awkward to hold.  this is our moment...

Reply:
Ok. I can post some ideas here before standup if that works.

Reply:
I've updated the schema for host_resolution. See the v4/host_resolution.json file.